File: mobile/iOS/GuestJini/GuestJini/Global/Model/Responses/GenericResponse.swift
//
//  GenericResponse.swift
//  GuestJini
//
//  Created by Prasanna Kumar Pete on 31/01/20.
//  Copyright © 2020 Prasanna Kumar Pete. All rights reserved.
//

import Foundation
class GenericResponse: Decodable {
    public var success:Bool = false
    public var error:String = ""
}

File: mobile/android/GuestJini/app/src/main/java/com/carmel/guestjini/Networking/GenericResponse.java
package com.carmel.guestjini.Networking;

public class GenericResponse {
    private Boolean success;
    private String error;

    public GenericResponse() {
        this.error = "";
        this.success = false;
    }

    public Boolean getSuccess() {
        return success;
    }

    public void setSuccess(Boolean success) {
        this.success = success;
    }

    public String getError() {
        return error;
    }

    public void setError(String error) {
        this.error = error;
    }
}
